fast-af/fast-checkout-woocommerce
# Resolves PINT-708 - Update product button option select color

# Description

This updates the "Select Product Button Location" option to make it more obvious which option is selected.

# Testing instructions

1. Login to the staging server.
2. Go to the Fast Settings page and click the Options tab.
3. Verify that the "Select Product Button Location" option is clear when try to determine which option is selected.

# Checks
